src/View/View.php

```diff
@@ -1255,11 +1255,14 @@ protected function _paths($plugin = null, $cached = true)
             }
         }
 
+        $class = new \ReflectionClass('\Cake\View\View');
+        $cake = dirname(dirname($class->getFilename()));
```

Comment:
I too think @ndm2 is right. 

@Phillaf maybe your problem is being cased by something else? How would one reproduce it?